app/Member.php

```diff
@@ -64,8 +64,9 @@ class Member extends Model
 
     public static $options_gender = array('Male', 'Female');
     public static $options_membership_type = array('Non-member', '1 year', 'Life');
-    public static $options_allowed_membership_type = array('1 year', 'Life');
+    public static $options_allowed_membership_type = array('1 year', 'Life', 'Life (Awaiting Payment)', '1 year (Awaiting Payment)');
```

Comment:
@tanhanliang @spencerthang The original design was for payment status to be incorporated via the booleans `is_fee_paid` and `is_card_issued`, thus distinguishing the concepts of Membership Type and the status of the membership application.